### ISSUE TITLE
Feature request: grid mesh on sphere

### ISSUE THREAD

**Issue by SpiritSuperUser** [spirit-code/spirit#447]
a grid mesh on the sphere visualization would be rather useful in my opinion

**Comment by GPMueller** [comment]
What do you mean by "on the sphere visualization"? Are you referring to what is done in this paper https://journals.aps.org/prb/abstract/10.1103/PhysRevB.93.214412 ?

**Comment by SpiritSuperUser** [comment]
no. i just meant that it would be nice to have a theta and phi grid shown on the sphere. for example to see easier where the poles are etc.
